tests/fw_rules_tests/policies/expected_output/calico-testcase14-scheme_output.csv

```diff
@@ -1,5 +1,5 @@
 "query","src_ns","src_pods","dst_ns","dst_pods","connection",
 "match-icmp-also-within-default-test, config: match-icmp-also-within-default","","","","","",
-"","[kube-system]","[app in (keepalived-watcher)]","[kube-system]","[app in (keepalived-watcher)]","ICMP [100-100] => [230];,",
+"","[kube-system]","[(app in (keepalived-watcher))]","[kube-system]","[(app in (keepalived-watcher))]","ICMP [100-100] => [230];,",
```

Comment:
Can we make it without the extra parenthesis?

Reply:
updated implementation so that these extra parenthesis are only added when required (e.g. at test7-scheme_output).